cmake/flash/CMakeLists.txt

```diff
@@ -186,7 +186,19 @@ foreach(target flash debug debugserver attach rtt)
     set(RUNNER_VERBOSE)
   endif()
 
-  if(WEST)
+  if(FLASH_COMMAND)
```

Comment:
Is there some place you can add some documentation/example on how to use the FLASH_COMMAND argument?